### PR TITLE
Bot blob

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
Add UI blob to DB

### CODE REVIEW

**Comment by kamilsmuga** [comment]
@tausifmuzaffar do you want to expose it via API in a separate PR?

File: db/model/bot.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ module.exports = function bot(seq, dataTypes) {
       validate: { isUrl: true },
       comment: 'The URL to load bot',
     },
+    ui: {
+      type: dataTypes.BLOB,
```
Comment (kamilsmuga) [inline db/model/bot.js]: Why binary data type?

Comment (tausifmuzaffar) [inline db/model/bot.js]: What you mean? This is the same pattern as lens.

**Comment by tausifmuzaffar** [comment]
@kamilsmuga API will be a seperate PR, but under the same user story W-4138685

File: db/model/bot.js (hunk 1 above)
Comment (kamilsmuga) [inline db/model/bot.js]: Clarified on standup. Looks good

File: tests/db/model/bot/utils.js [hunk 2]
```diff
@@ -10,16 +10,20 @@
...
 const testStartTime = new Date();
 const n = `${tu.namePrefix}TestBot`;
 const n2 = n+'NonActive';
+const mt = path.join(__dirname, './uiBlob');
+const uiBlob = fs.readFileSync(mt);
 
 const standard = {
   name: n,
   url: 'http://www.bar.com',
+  ui: uiBlob,
```
Comment (pallavi2209) [inline tests/db/model/bot/utils.js]: An update db test for this field would be great, but other than that, I am fine merging this PR.

File: tests/db/model/bot/update.js [hunk 3]
```diff
@@ -48,6 +52,17 @@ describe('db: bot: update: ', () => {
       .catch(done);
     });
 
+    it('ok, bot ui updated', (done) => {
+      Bot.findOne({ where: { name: u.name } })
+      .then((o) => o.update({ url: 'http://www.test.com' }))
```
Comment (pallavi2209) [inline tests/db/model/bot/update.js]: ui instead of url

**Pull request merged**